.github/workflows/ci.yml

```diff
@@ -155,6 +155,24 @@ jobs:
             exit $LASTEXITCODE
           }
 
+      - name: Validate tools/go.mod
+        shell: powershell
+        working-directory: tools
+        run: |
+          Write-Output "::group::go mod tidy"
+          go mod tidy
```

Comment:
Does this not need a `go mod vendor`?

Reply:
nope, since (like `test`), were not vendoring anything outside the main repo